ADDING_LEARN_PAGE.md

```diff
@@ -0,0 +1,44 @@
+# Documentation RFCs (Request for Comments)
+
+While the Website Redesign working group continues to iterate on site design and content structure, we will use an RFC process to accept and evaluate Node.js documentation contributions in this repo. Once the new site architecture and documentation ingestion flow is finalized, all docs content will be migrated to its final location and this process updated accordingly.
+
+## When you need to follow this process
+
+With the working group iterating on design and content structure, this process is our way of making sure all content gets good feedback and, with approval, eventually finds its way to the right place in the documentation.
+
+You need to follow this process if you intend to make a "substantial" addition or change to the proposed Node.js getting started guides in this repo. What constitutes a "substantial" change evolves based on community norms. Some changes do not require an RFC. These may include reasonably sized re-phasing, reorganizing, or refactoring an existing documentation page.
+
+If you submit a pull request to implement a significant change to the documentation without going through the RFC process, it may be closed with a polite request to submit an RFC first!
+
+## To begin drafting a documentation page:
+
+1. Fork the [nodejs/website-redesign repo](https://github.com/nodejs/website-redesign) so that you have your own copy of the repository that you have push access to.
```

Comment:
```suggestion
1. Fork the [nodejs/nodejs.dev repo](https://github.com/nodejs/nodejs.dev) so that you have your own copy of the repository to which you have push access.
```